velox/docs/configs.rst

```diff
@@ -399,6 +399,26 @@ Spilling
      - 0
      - Percentage of aggregation or join input batches that will be forced to spill for testing. 0 means no extra spilling.
 
+Aggregation
```

Comment:
Thank you for starting a new section. Would you move abandon_partial_aggregation_min_rows and other agg-specific config here?